driver/registry_base.go

```diff
@@ -432,11 +432,20 @@ func (m *RegistryBase) SubjectIdentifierAlgorithm() map[string]consent.SubjectId
 
 func (m *RegistryBase) Tracer(ctx context.Context) *tracing.Tracer {
 	if m.trc == nil {
-		t, err := tracing.New(m.l, m.C.Tracing())
-		if err != nil {
-			m.Logger().WithError(err).Fatalf("Unable to initialize Tracer.")
+		var err error
+		var t *tracing.Tracer
+
+		// Make 3 attempts to initialize tracer
+		for i := 0; i < 3; i++ {
```

Comment:
Good idea! Unfortunately, this doesn't wait so it will try 3 times in rapid succession and failing. I think it's ok to just print the error with `Error` to the logger without retries!

Reply:
I was meaning to put a `time.Sleep()` at the end of each iteration, but was unsure of how long it should sleep. Would you suggest I do that or do I get rid of the retries entirely?